shared/utils/enable-centos.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/python2
+
+"""
+Takes given XCCDF or DataStream and adds CentOS CPE name next to RHEL CPE names.
+Can automatically recognize RHEL5, 6, 7 CPEs and adds the CentOS ones next to
+those accordingly.
+
+Apart from adding the CPEs it adds a notice informing the user that the content
+has been enabled for CentOS and what are the implications.
+
+Author: Martin Preisler <mpreisle@redhat.com>
+"""
+
+from xml.etree import cElementTree as ElementTree
+import sys
+
+XCCDF11_NS = "http://checklists.nist.gov/xccdf/1.1"
+XCCDF12_NS = "http://checklists.nist.gov/xccdf/1.2"
+
+RHEL_CENTOS_CPE_MAPPING = {
+    "cpe:/o:redhat:enterprise_linux:4": "cpe:/o:centos:centos:4",
+    "cpe:/o:redhat:enterprise_linux:5": "cpe:/o:centos:centos:5",
+    "cpe:/o:redhat:enterprise_linux:6": "cpe:/o:centos:centos:6",
+    "cpe:/o:redhat:enterprise_linux:7": "cpe:/o:centos:centos:7",
+}
+
+CENTOS_NOTICE = \
+    "<div xmlns=\"http://www.w3.org/1999/xhtml\">\n" \
+    "<p>This benchmark is a direct port of a <i>SCAP Security Guide </i> " \
+    "benchmark developed for <i>Red Hat Enterprise Linux</i>. It has been " \
+    "modified through an automated process to remove specific dependencies " \
+    "on <i>Red Hat Enterprise Linux</i> and to function with <i>CentOS</i> " \
+    "&#8212; <i>Community Enterprise Operating System</i>. The result is a " \
+    "generally useful <i>SCAP Security Guide</i> benchmark with the following " \
+    "caveats:</p>\n" \
+    "<ul>\n" \
+    "<li><i>CentOS</i> is not an exact copy of " \
+    "<i>Red Hat Enterprise Linux</i>. There may be configuration differences " \
+    "that produce both false positives and false negatives. If this occurs " \
+    "please file a bug report.</li>\n" \
+    "\n" \
+    "<li><i>CentOS</i> has its own build system, compiler options, patchsets, " \
+    "and is a community-led operating system, <i>CentOS</i> does not inherit " \
```

Comment:
Within many regulated environments, policies exist to favor COTS software. It'd be good to gently remind users that CentOS is not commercial software nor commercially supported. How about:

````.... and is a community supported, non-commercial operating system. <i>CentOS</i> does not....````

Note the new period too.